.github/workflows/check.yml

```diff
@@ -83,11 +83,11 @@ jobs:
     - name: Build flatpak
       run: ninja -C _build
       env:
-        ASAN_OPTIONS: detect_leaks=0 # Right now we're not fully clean, but this gets us use-after-free etc
+        ASAN_OPTIONS: detect_leaks=1
```

Comment:
Probably just remove the env var entirely since its the default.

Reply:
yeah, but I just wanted to test for now. there are other issues to tackle.